Makefile

```diff
@@ -18,40 +18,40 @@ endif
 container ?= web
 
 build:
-	$(docker-compose) run web bundle install
-	$(docker-compose) run web yarn install
 	$(docker-compose) build
-	$(docker-compose) run web rails db:migrate
+	$(docker-compose) run --rm web bundle install -j $(shell expr $(shell getconf _NPROCESSORS_ONLN) - 1)
```

Comment:
By adding `--rm` to these commands, after the command exits it will no longer leave an unnamed image floating around taking up space until you prune dangling images.